doc/user-guide/registration_service.md

```diff
@@ -0,0 +1,59 @@
+## Registration service
+
+Current release Indigo IAM introduces the functionality that allows users to submit a request of membership
+to Indigo.
+
+#### New registration
+
+For insert a new registration request, users can go to IAM login page and click the _Register a new account_ button.
+
+![INDIGO IAM Registration button](../images/IAM-registration-1.png)
+
+Name, surname, email and username are mandatory fields. With _notes_ field, users can tell to the IAM administrators any extra information about their request of membership.
+
+![INDIGO IAM Registration form](../images/IAM-registration-2.png)
+
+#### Confirm registration request
+
+If the submission ends successfully, an email with a verification link is sent to the user for 
+confirm his registration request and "validate" his email address.
+
+![INDIGO IAM Registration validation page](../images/IAM-registration-3.png)
+
+When a new request is confirmed, IAM administrators are notified with an email too.
+
+#### Requests approval or rejection 
+
+Only users with role `ROLE_ADMIN` or `registration:write` scope, can decide to approve or reject a registration request.
+To performe this operation, log into IAM as administrator and go to the new dashboard.
```

Comment:
go to the new dashboard -> open the new dashboard by clicking the "New Admin Dashboard" button